recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
:+1: if you don't mind having `h5py` pinned to `hdf5 1.8.15.1`! That will solve the `xarray` issue.